src/validations/aave-space-validation/index.ts

```diff
@@ -0,0 +1,56 @@
+import { call, getScores } from '../../utils';
```

Comment:
The `call` function doesn't seem to be used you can remove it
```suggestion
import { getScores } from '../../utils';
```